apps/public-data/src/public-data/public-data.module.ts

```diff
@@ -1,13 +1,27 @@
+import { User } from '@apps/user/src/user/entities/user.entity';
 import { Module } from '@nestjs/common';
 import { ClientsModule, Transport } from '@nestjs/microservices';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { PublicDataController } from './public-data.controller';
 import { PublicDataService } from './public-data.service';
-import { PostCategory } from './entities/post-category.entity';
+import {
+  PostCategory,
+  CareerCategory,
+  PositionCategory,
+  ProjectCategory,
+  StackCategory,
+} from './entities';
 
 @Module({
   imports: [
-    TypeOrmModule.forFeature([PostCategory]),
+    TypeOrmModule.forFeature([
+      PostCategory,
+      PositionCategory,
+      CareerCategory,
+      StackCategory,
+      User,
+      ProjectCategory,
+    ]),
```

Comment:
사소하지만 요렇게 정렬해줘도 괜찮을 것 같습니다~
외부 모듈 -> 내부 모듈 + 같은 역할을 하는 것 끼리
```suggestion
    TypeOrmModule.forFeature([
      User,
      PostCategory,
      PositionCategory,
      CareerCategory,
      StackCategory,
      ProjectCategory,
    ]),
```